File: app_patine/lib/app/domain/entities/exercicio.dart
//@dart=2.9

class Exercicio {
  dynamic id;
  String nome;
  String descricao;
  String urlImagem;
  int likes;
  int dislikes;
  //dynamic treinadorId;

  Exercicio(
      {this.id,
      this.nome,
      this.descricao,
      this.urlImagem,
      this.likes = 0,
      this.dislikes = 0
      //this.treinadorId
      });
}

File: patineapi/patinerestapi/src/main/java/com/patine/patinerestapi/entidade/Exercicio.java
package com.patine.patinerestapi.entidade;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

@Entity
public class Exercicio {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(nullable = false)
    private String nome;

    @Column(nullable = false)
    private String descricao;


    private String urlImagem;


    public Long getId() {
        return id;
    }


    public void setId(Long id) {
        this.id = id;
    }


    public String getNome() {
        return nome;
    }


    public void setNome(String nome) {
        this.nome = nome;
    }


    public String getDescricao() {
        return descricao;
    }


    public void setDescricao(String descricao) {
        this.descricao = descricao;
    }


    public String getUrlImagem() {
        return urlImagem;
    }


    public void setUrlImagem(String urlImagem) {
        this.urlImagem = urlImagem;
    }

    
    
}
